### ISSUE TITLE
Preserve Content-Disposition filename

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#2815]
When downloading from a generic address (e.g. "download"), the Content-Disposition header may have a better filename to use as the starting point for the project than the URL. In the example from #2593, https://usace.contentdm.oclc.org/digital/api/collection/p16021coll2/id/4706/download 
has

`Content-Disposition: attachment; filename="p16021coll2_4706.xlsx"`

which both gives us a better name and a hint at the importer to use.

**Comment by thadguidry** [comment]
@tfmorris Ah.
1. Hmm, what about `filename*` as well?
>**filename***
> The parameters "filename" and "filename*" differ only in that "filename*" uses the encoding defined in RFC 5987. When both "filename" and "filename*" are present in a single header field value, "filename*" is preferred over "filename" when both are understood.


REF: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

**Comment by tfmorris** [comment]
> what about filename* as well?

Only if whatever library we end up using supports rfc5987 out of the box

**Comment by antoine2711** [comment]
@tfmorris: I don't think this is working. When I try to download a .CSV file, I get a new tab and not a download.

Regards,
   Antoine

**Comment by tfmorris** [comment]
> I don't think this is working. When I try to download a .CSV file, I get a new tab and not a download.

@antoine2711 Are you talking about a different issue than this one? This is still open and doesn't have anything to do with downloading to a browser.

**Comment by antoine2711** [comment]
> > I don't think this is working. When I try to download a .CSV file, I get a new tab and not a download.
> 
> @antoine2711 Are you talking about a different issue than this one? This is still open and doesn't have anything to do with downloading to a browser.

@tfmorris: which Issue or PR would be the good one? 

The behavior seams very different from before. Anyway, shouldn't a request to download a file just do that, and not open a new tab with the content? Also, there is something weird, my Chrome CAN'T save the page, even is it's just text.

Regards,
   Antoine

**Comment by tfmorris** [comment]
@antoine If you can't find a relevant issue (this definitely isn't it!), please create a new one. Please include the version that you are testing (3.4-beta1, 3.4-head or master?), what function(s) you used (Export to CSV? Custom Tabular Export? something else?), the expected vs actual behavior, etc. In other words, all the usual stuff that you would put in a high quality bug report.
